test/useTranslation.loading.spec.js

```diff
@@ -49,73 +15,32 @@ describe('useTranslation loading ns', () => {
     newI18n.use(backend).init({
       lng: 'en',
       fallbackLng: 'en',
-
       interpolation: {
         escapeValue: false, // not needed for react!!
       },
     });
   });
 
-  const TestElement = ({ useSuspense = true }) => {
-    const [t] = useTranslation('common', { i18n: newI18n, useSuspense });
-    return <div>{t('key1')}</div>;
-  };
-
-  it('should wait for correct translation', done => {
-    const spy = sinon.spy();
-
-    console.error = jest.fn(); // silent down the error boundary error from react-dom
-    let wrapper = mount(
-      <ErrorBoundary spy={spy}>
-        <TestElement />
-      </ErrorBoundary>,
+  it('should wait for correct translation RTL', async () => {
```

Comment:
shouldn't this be `'should wait for correct translation'`? (without RTL)

Reply:
Jupp :partying_face:. I see now, one with suspense and one without :tada: